Makefile

```diff
@@ -113,7 +113,7 @@ test-unit: start-test-redis
 		--link ${REDIS_CONTAINER_NAME}:TEST_REDIS \
 		${DEV_ENV_IMAGE} bash -c 'DEIS_LOGGER_REDIS_SERVICE_HOST=$$TEST_REDIS_PORT_6379_TCP_ADDR \
 		DEIS_LOGGER_REDIS_SERVICE_PORT=$$TEST_REDIS_PORT_6379_TCP_PORT \
-		$(GOTEST) $$(glide nv)' \
+		$(GOTEST) -tags="testredis" $$(glide nv)' \
```

Comment:
nope, this just includes the file tagged with `testredis` into the build and test. here's a snippet from test output:

```console
...
docker run --rm -v /Users/aaronschlesinger/gocode/src/github.com/deis/logger:/go/src/github.com/deis/logger -w /go/src/github.com/deis/logger \
		-it \
		--link test-redis-devel:TEST_REDIS \
		quay.io/deis/go-dev:0.13.0 bash -c 'DEIS_LOGGER_REDIS_SERVICE_HOST=$TEST_REDIS_PORT_6379_TCP_ADDR \
		DEIS_LOGGER_REDIS_SERVICE_PORT=$TEST_REDIS_PORT_6379_TCP_PORT \
		go test --cover --race -v -tags="testredis" $(glide nv)' \
		|| (make stop-test-redis && false)
=== RUN   TestGetUsingInvalidValues
--- PASS: TestGetUsingInvalidValues (0.00s)
=== RUN   TestNSQBasedAggregator
--- PASS: TestNSQBasedAggregator (0.00s)
PASS
coverage: 6.8% of statements
ok  	github.com/deis/logger/log	1.010s	coverage: 6.8% of statements
=== RUN   TestFactoryGetUsingInvalidValues
--- PASS: TestFactoryGetUsingInvalidValues (0.00s)
=== RUN   TestFactoryGetFileBasedAdapter
--- PASS: TestFactoryGetFileBasedAdapter (0.00s)
=== RUN   TestFactoryGetMemoryBasedAdapter
--- PASS: TestFactoryGetMemoryBasedAdapter (0.00s)
=== RUN   TestGetRedisBasedAdapter
--- PASS: TestGetRedisBasedAdapter (0.00s)
=== RUN   TestReadFromNonExistingApp
--- PASS: TestReadFromNonExistingApp (0.00s)
=== RUN   TestLogs
--- PASS: TestLogs (0.00s)
=== RUN   TestDestroy
--- PASS: TestDestroy (0.00s)
=== RUN   TestReopen
--- PASS: TestReopen (0.00s)
=== RUN   TestRedisReadFromNonExistingApp
--- PASS: TestRedisReadFromNonExistingApp (0.00s)
=== RUN   TestRedisWithBadBufferSizes
--- PASS: TestRedisWithBadBufferSizes (0.00s)
=== RUN   TestRedisLogs
--- PASS: TestRedisLogs (0.00s)
=== RUN   TestRedisDestroy
--- PASS: TestRedisDestroy (0.00s)
=== RUN   TestRingBufferReadFromNonExistingApp
--- PASS: TestRingBufferReadFromNonExistingApp (0.00s)
=== RUN   TestRingBufferWithBadBufferSizes
--- PASS: TestRingBufferWithBadBufferSizes (0.00s)
=== RUN   TestRingBufferLogs
2016/07/01 14:48:18 Creating buffer for app:test-app
--- PASS: TestRingBufferLogs (0.00s)
=== RUN   TestRingBufferDestroy
2016/07/01 14:48:18 Creating buffer for app:test-app
--- PASS: TestRingBufferDestroy (0.00s)
PASS
...
```